public/workers/addressDiscovery.js

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const { ExplorerClient, walletImport, discoverActiveAddresses } = require('@alephium/sdk')
```

Comment:
Can't a web worker be written in typescript?

Reply:
Looks like someone managed to do it using `react-scripts v5` with a gotcha that there's a workaround to make jest work:

- https://github.com/facebook/create-react-app/issues/11651
- https://github.com/cmdcolin/cra-webpack5-web-worker-example

Reply:
Maybe we should consider using Vite on this project as well?

Reply:
Yes 👍 #445